src/client/index.js

```diff
@@ -608,3 +613,33 @@ $('[data-bs-toggle="tooltip"]').tooltip();
 
 });
 
+
+let isAnyTabOpen = false;
```

Comment:
You never set this back to `false`. Is this needed? Why not have one of the tabs always opened, even though it's not visible in "desktop" mode?

Reply:
Actually yes - I could initially open one, arbitrary selected tab.
But the flag is needed for another purpose - on window resize we want to open the tab, that should be open.

We surely don't want to change the tab to the "default one" on every resize when screen is less than 925px wide.
Here we check it to prevent this behavior:
`window.onresize = () => {
    if (window.innerWidth <= 925 && !isAnyTabOpen) {
        openTab(codeTabClass);
    }
};`

Reply:
Why do you want to open a tab when resizing? Why can't it be already open? This way you could throw away the entire onresize callback.

Reply:
Yes you were right... I noticed it after a while of thinking what an overkill this solution was. I simplified it in the last commit.

Is it good now?